generators/writing/templates/src/services/_service/name.service-mongodb.ejs

```diff
@@ -37,6 +38,7 @@
 
   service.hooks(hooks)<%- sc %>
   <%- insertFragment('func_return') %>
+  registerServicePromise(app, promise)
```

Comment:
@eddyystop I've put this after the `func_return` so that the user can modify the promise if they so desire.

Reply:
That's the idea!